Header header
sensor_msgs/PointCloud2 fullPC
sensor_msgs/PointCloud2 CornerPC
sensor_msgs/PointCloud2 SurfPC
nav_msgs/Odometry pose
nav_msgs/Odometry increment
int32 frame_idx
